sources/packages/web/src/components/layouts/Institution/sidebar/HomeSideBar.vue

```diff
@@ -1,14 +1,20 @@
 <template>
-  <v-navigation-drawer app class="body-background" permanent>
-    <v-list density="compact" nav>
+  <v-navigation-drawer app v-model="drawer" color="background" permanent>
+    <v-list
+      active-class="active-sidebar-item"
+      density="compact"
+      bg-color="background"
+      active-color="primary"
+    >
       <v-list-item
         v-for="item in items"
         :key="item.label"
-        @click="item.command"
+        :to="item.command()"
         :prepend-icon="item.icon"
         :title="item.label"
+        :value="item.value"
```

Comment:
do we have value as property in item?

Reply:
yup value prop can be added with all components or parent components with v-model. but here it's not required, it's a leftover removing.